core/src/main/scala/kafka/server/ConfigAdminManager.scala

```diff
@@ -515,4 +512,16 @@ object ConfigAdminManager {
       }
     }
   }
+
+  def getConfigPropertyAsList(
+    configProps: Properties,
+    configKeys: Map[String, ConfigKey],
+    configPropName: String
+  ): List[String] = {
+    Option(configProps.getProperty(configPropName))
+      .orElse(Option(ConfigDef.convertToString(configKeys(configPropName).defaultValue, ConfigDef.Type.LIST)))
+      .filter(_.trim.nonEmpty)
+      .map(_.split(",").toList)
```

Comment:
Do you think that we should trim and filter after splitting based on commas? For example should `"first , second, third"` be equivalent to `"first,second,third"`?